### DISCUSSION TITLE
Regarding Model Deployment

### DISCUSSION THREAD

**Discussion by jalaygit** [ultralytics/ultralytics#13018] [Q&A]
Hello,

I have developed and trained my Model using YoloV8 for my project, I am able to get results in Ultralytics HUB on the preview bar. But now I wan to deploy my model in Raspberry Pi 5.
Kindly help me with the steps to do so.

Thanking in advance

**Comment by Asshok1** [comment]
@jalaygit  I think it is better to deploy your model in Jetson Kit . Raspberry Pi are slow, they provide delay in output when compared with Jetson kits
